apps/src/templates/projects/PublicGallery.jsx

```diff
@@ -42,18 +33,22 @@ class PublicGallery extends Component {
    * See the PropTypes of each component for a definition of each format.
    */
   mapProjectData(projectLists) {
-    return _.mapValues(projectLists, projectList => {
-      return projectList.map(projectData => {
-        return {
+    let allFeaturedProjects = [];
+    for (const projectListName in projectLists) {
+      projectLists[projectListName].forEach(projectData => {
+        allFeaturedProjects.push({
```

Comment:
You don't need to change this, but as a suggestion - 

two things jump out - one is that I just don't care for mixing explicit loops and methods. So either it should be two nested for loops or two nested `forEach` loops.

But moreso, it's good to get rid of using mutable variables defined externally outside of loops/callbacks, where possible. I think this'll do the same thing for you:

```
const allFeaturedProjects = projectLists.flatMap(
  projectListName => projectListName.flatMap(
    projectData => (
    {
      projectData: {
        ...projectData,
        publishedToPublic: true,
        publishedToClass: false,
      },
      currentGallery: 'public',
    }
  )
);
```

Reply:
Thanks Jim! `projectLists` is not an array but an object so I first create an array or project data objects and then transform the data. I also updated the comment to hopefully better clarify this function.